rstudio/rstudio
# Don't localize "untitled" for new doc filenames

### Intent

Addresses #10837

Don't localize the name of new files, always use English "Untitled".

### Approach

Put back hardcoded "Untitled" in relevent places.

### Automated Tests

None, we don't have automation for localized UI and the English behaviour doesn't change.

### QA Notes

Try creating new documents in a variety of ways with UI in French and confirm they are "Untitled" not "Sans Titre".

### Checklist
